jormungandr-lib/src/interfaces/settings.rs

```diff
@@ -14,6 +21,53 @@ pub struct SettingsDto {
     pub max_txs_per_block: u32,
     pub slot_duration: u64,
     pub slots_per_epoch: u32,
+    #[serde(with = "TaxTypeDef")]
+    pub treasury_tax: TaxType,
+    #[serde(with = "ParametersDef")]
+    pub reward_params: Parameters,
+}
+
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(deny_unknown_fields, remote = "TaxType")]
+pub struct TaxTypeDef {
+    #[serde(with = "ValueDef")]
+    pub fixed: Value,
+
+    #[serde(with = "RatioDef")]
+    pub ratio: Ratio,
+
+    #[serde(default, rename = "max", skip_serializing_if = "Option::is_none")]
+    pub max_limit: Option<NonZeroU64>,
+}
+
+#[derive(Serialize)]
+#[serde(transparent)]
+pub struct TaxTypeSerde(#[serde(with = "TaxTypeDef")] pub TaxType);
+
+#[derive(Debug, Clone, Copy, Deserialize, Serialize, PartialEq, Eq)]
+#[serde(remote = "Ratio")]
+pub struct RatioDef {
+    pub numerator: u64,
+    pub denominator: NonZeroU64,
+}
+
+#[derive(Deserialize, Serialize)]
+#[serde(remote = "Parameters", rename_all = "camelCase")]
+pub struct ParametersDef {
+    pub initial_value: u64,
+    #[serde(with = "RatioDef")]
+    pub compounding_ratio: Ratio,
```

Comment:
duplicate of https://github.com/input-output-hk/jormungandr/blob/9a10bf7bc211969e8c331532b956f6e4110d9446/jormungandr-lib/src/interfaces/ratio.rs#L12

Reply:
Same as above